src/locales/cy/translation.json

```diff
@@ -264,23 +266,99 @@
         },
         "updateUserDetailsInformation": {
           "updateDetailsLabel": "Sut i ddiweddaru eich manylion",
-          "updateDetailsHtml": "Gallwch <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://signin.account.gov.uk/contact-us?supportType=PUBLIC\">gysylltu â’r tîm GOV.UK One Login (agor mewn tab newydd)</a> i ddiweddaru eich manylion."
+          "updateDetailsHtml": "Cysylltwch â thîm cymorth <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://signin.account.gov.uk/contact-us?supportType=PUBLIC\">GOV.UK One Login (agor mewn tab newydd)</a>."
         }
       }
     },
     "pageIpvPending": {
       "title": "Nid ydych wedi gorffen profi eich hunaniaeth",
       "header": "Nid ydych wedi gorffen profi eich hunaniaeth",
       "content": {
-        "paragraph1": "Mae angen i chi fynd i Swyddfa'r Post sy'n cynnig dilysu mewn canghennau i orffen profi eich hunaniaeth.",
-        "paragraph2": "Dylech fod wedi derbyn e-bost cadarnhau yn esbonio beth i fynd gyda chi a beth fydd yn digwydd yn Swyddfa'r Post. Gwiriwch y llythyr yn eich e-bost cadarnhau.",
-        "subHeading": "Os ydych eisoes wedi bod i Swyddfa'r Post",
+        "paragraph1": "Mae angen i chi fynd i Swyddfa’r Post sy’n cynnig dilysu mewn canghennau i orffen profi eich hunaniaeth.",
+        "paragraph2": "Dylech fod wedi derbyn e-bost cadarnhau yn esbonio beth i fynd gyda chi a beth fydd yn digwydd yn Swyddfa’r Post. Gwiriwch y llythyr yn eich e-bost cadarnhau.",
+        "subHeading": "Os ydych eisoes wedi bod i Swyddfa’r Post",
         "paragraph3": "Mae eich gwiriad hunaniaeth yn dal i gael ei brosesu.",
-        "paragraph4": "Byddwch yn cael e-bost pan fydd canlyniad i'ch gwiriad hunaniaeth - fel arfer o fewn diwrnod o fynd i Swyddfa'r Post.",
+        "paragraph4": "Byddwch yn cael e-bost pan fydd canlyniad i’ch gwiriad hunaniaeth – fel arfer o fewn diwrnod o fynd i Swyddfa’r Post.",
+        "subHeading2": "Os ydych angen help",
+        "paragraph5": "Contact us if you:",
+        "list1": "need help to finish proving your identity with the Post Office",
+        "list2": "would prefer not to go to the Post Office and want to prove your identity another way"
```

Comment:
The Welsh for this has just been merged- https://github.com/alphagov/di-ipv-core-front/pull/956/files
Conflict gone wrong maybe? might be easier to just copy it over